Cubical/ZCohomology/Properties.agda

```diff
@@ -199,6 +200,47 @@ leftInv (Iso-coHom-coHomRed {A = A , a} n) =
                                   ; (j = i1) → 0ₖ (2 + n)})
                         (0ₖ (2 + n))
 
++∙≡+ : (n : ℕ) {A : Pointed ℓ} (x y : coHomRed (suc n) A)
+     → Iso.fun (Iso-coHom-coHomRed n) (x +ₕ∙ y)
+      ≡ Iso.fun (Iso-coHom-coHomRed n) x +ₕ Iso.fun (Iso-coHom-coHomRed n) y
+
++∙≡+ zero = sElim2 (λ _ _ → isOfHLevelPath 2 § _ _) λ _ _ → refl
++∙≡+ (suc n) = sElim2 (λ _ _ → isOfHLevelPath 2 § _ _) λ _ _ → refl
+
+private
+  homhelp : ∀ {ℓ} (n : ℕ) (A : Pointed ℓ) (x y : coHom (suc n) (typ A))
+          → Iso.inv (Iso-coHom-coHomRed {A = A} n) (x +ₕ y)
+           ≡ Iso.inv (Iso-coHom-coHomRed n) x +ₕ∙ Iso.inv (Iso-coHom-coHomRed n) y
+  homhelp n A = morphLemmas.isMorphInv _+ₕ∙_ _+ₕ_
+                (Iso.fun (Iso-coHom-coHomRed n)) (+∙≡+ n) _
+                (Iso.rightInv (Iso-coHom-coHomRed n)) (Iso.leftInv (Iso-coHom-coHomRed n))
+
+coHomGr≅coHomRedGr : ∀ {ℓ} (n : ℕ) (A : Pointed ℓ)
+                  → GroupEquiv (coHomRedGrDir (suc n) A) (coHomGr (suc n) (typ A))
+GroupEquiv.eq (coHomGr≅coHomRedGr n A) = isoToEquiv (Iso-coHom-coHomRed n)
+GroupEquiv.isHom (coHomGr≅coHomRedGr n A) = +∙≡+ n
+
+coHomGr≡coHomRedGr : ∀ {ℓ} (n : ℕ) (A : Pointed ℓ)
+                  → coHomRedGrDir (suc n) A ≡ coHomGr (suc n) (typ A)
+coHomGr≡coHomRedGr n A = uaGroup (coHomGr≅coHomRedGr n A)
+
+coHomGroup≡coHomRedGroup' : ∀ {ℓ} (n : ℕ) (A : Pointed ℓ)
```

Comment:
Shouldn't this be local to the lemma below?

Reply:
I'm using it twice below, so that won't work really. If I understand what you're saying correctly, that is:-)